capabilities/learning-culture.md

```diff
@@ -42,14 +47,19 @@ Leading workshops is an excellent method for providing participants with hands-o
 
 Hosting a roundtable discussion involves coming prepared with a few key talking points, then allowing the conversation to evolve organically. A good host ensures that the discussion remains productive while limiting their interventions, welcoming productive tangents that can lead to deeper insights. This format is particularly effective when the topic is already somewhat understood by the participants, as it encourages a deeper understanding through collaborative dialogue and diverse perspectives. Roundtable discussions are excellent for spurring meaningful conversations and uncovering new angles on familiar topics.
 
-### [Start A Book Club](/practices/start-a-book-club.md)
 
-Starting a book club can encompass a variety of materials, including books, whitepapers, blogs, articles, and podcasts. This practice is excellent for encouraging experienced practitioners to consider new ideas and perspectives. By engaging with diverse content and participating in thoughtful discussions, members can expand their knowledge and challenge their existing viewpoints. Book clubs provide a structured yet flexible environment for continuous learning and professional growth. However, if there is a lot of content to go through, this burden can create drop-off for less invested parties. This needs to be managed.
+## Getting Your Team Started
+Help your team implement this capability by hosting a viewing party of specially curated talks, starting a book club to discuss impactful titles on this topic, or running a timeboxed experiment (or spike).
+
```

Comment:
I'm suggesting we add this head here, since some of the items that follow aren't true "Practices." For example, Start a Book Club is listed under Getting Your Team Started in various Practice documents. But it itself is not a Practice. 

Another way to handle this is to not have this section but to retitle the "Supporting Practices" header "Supporting Practices and Strategies".

Reply:
Here's our definition of practice from the main readme: `An actionable pattern, technique, or process employed by software professionals.` I feel like all of these are practices. It's true that we use some of these practices as how to improve categories for other practices. I view them as higher level practices, but practices nonetheless. Should we schedule a time to discuss this? I don't want to dismiss your suggestion to re-organize this somehow. I don't want to accept the change as it currently stands. Perhaps there's something here that a conversation could extract??

Reply:
OK, this makes sense to me. I'd like to propose the following: 

- I stet (undo) my edits to this section here (no adding Getting Your Team Started). 
- We simplify the language we use for all Getting Your Team Started sections (which only appear in Practices) with: "The following are actionable patterns, techniques, or processes that will help your team implement this practice."